core/src/test/java/io/cucumber/core/filter/TagPredicateTest.java

```diff
@@ -3,19 +3,22 @@
 import io.cucumber.core.feature.TestFeatureParser;
 import io.cucumber.core.gherkin.Feature;
 import io.cucumber.core.gherkin.Pickle;
+import io.cucumber.tagexpressions.TagExpressionParser;
 import org.junit.jupiter.api.Test;
 
 import static java.util.Arrays.asList;
 import static java.util.Collections.singletonList;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 class TagPredicateTest {
 
     @Test
     void empty_tag_predicate_matches_pickle_with_any_tags() {
         Pickle pickle = createPickleWithTags("@FOO");
-        TagPredicate predicate = new TagPredicate("");
+        TagPredicate predicate = new TagPredicate(TagExpressionParser.parse(""));
```

Comment:
Might be good to extract a little helper method here that takes a vararg of strings and returns a lsit of tag expressions. Avoids some repetition.

Reply:
Cool.  I will have a look at this to see if I can do it.